string taskName
string startTime #ex 15:24 means 3:24PM
string taskType #{'medicine','lunch','dummytask'}
string parameters
